### ISSUE TITLE
Week 25/03/2024

### ISSUE THREAD

**Issue by olivierlabayle** [TARGENE/targene-pipeline#158]
Learning:
- [x] TMLE lecture / Semi parametrics / Causal Inference reading
- [x] Read one paper from Ideas: https://github.com/TARGENE/targene-pipeline/issues/97

VDR Study:
- [ ] Rerun with new targene release and possibly updated Q learning config
- [x] Analyse perm tests

Simulation Study:
- [x] Fix TMPDIR issue (probably just write to local dior instead)
- [x] [Density Estimation Simulation](https://github.com/TARGENE/targene-pipeline/issues/154)
- [x] Checkpoint best neural net
- [x] Add retrain on whole data in model selection
- [x] Find good heuristic to increase model size
- [x] Add some architecture twists: e.g. batchnorm
- [ ] Make sieve neural net and GLM estimators
- [x] Test quality of sample
- [ ] [Analysis](https://github.com/TARGENE/targene-pipeline/issues/155)
- [x] Write Configuration File for permutation null sampler
- [x] Write Configuration File for Density estimation model comparison
- [x] Write COnfiguration File for Density Estimation Sampler estimation
- [ ] Define study scope

Post PhD:
- [ ] Rework Resume
- [x] Ask for referee
